.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Enhancement 💡
+about: Suggest an enhancement for the YAML Language Server
+title: ''
+labels: 'kind/enhancement'
+assignees: ''
+
+---
+### Is your enhancement related to a problem? Please describe.
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
+
+### Describe the solution you'd like
```

Comment:
contractions are harder for non-native speakers to translate, please consider breaking it apart